File: app/src/main/java/player/phonograph/interfaces/CabHolder.java
package player.phonograph.interfaces;

import androidx.annotation.NonNull;

import com.afollestad.materialcab.MaterialCab;

/**
 * @author Karim Abou Zeid (kabouzeid)
 */
public interface CabHolder {

    @NonNull
    MaterialCab showCab(final int menuRes, final MaterialCab.Callback callback);
}

File: app/src/main/java/player/phonograph/interfaces/CabHolder.kt
package player.phonograph.interfaces

import com.afollestad.materialcab.CreateCallback
import com.afollestad.materialcab.DestroyCallback
import com.afollestad.materialcab.SelectCallback
import com.afollestad.materialcab.attached.AttachedCab

/**
 * @author Karim Abou Zeid (kabouzeid)
 */
interface CabHolder {
    fun showCab(menuRes: Int, createCallback: CreateCallback, selectCallback: SelectCallback, destroyCallback: DestroyCallback): AttachedCab
}
